app-nextjs-starter/src/styles/global.css

```diff
@@ -1,40 +1,24 @@
 html {
-  font-size: 10px;
+  height: 100%;
 }
 
-html,
 body {
-}
-
-body {
-  font-size: 1.4rem;
+  font-size: 0.875rem;
   padding: 0;
   margin: 0;
-  font-family: Roboto, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
-  Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+  font-family: Roboto, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto,
+    Oxygen, Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
   line-height: 1.6;
   color: #1b243f;
+  height: 100%;
 }
 
-* {
-  box-sizing: border-box;
-  -webkit-font-smoothing: antialiased;
+body > div {
+  height: 100%;
+  display: flex;
 }
```

Comment:
Is this targeting the root node where the next app mounts? (`#app` or whatever it's called)

What if there are multiple div elements inside body? For example if inserted by a browser extension?